tests/components/twitch/test_twitch.py

```diff
@@ -0,0 +1,174 @@
+"""The tests for an update of the Twitch component."""
```

Comment:
Please remove twitch/sensor.py from the omit section of .coveragerc.

Reply:
Removed :)